common/scripted_triggers/00_cultural_triggers.txt

```diff
@@ -209,8 +209,19 @@ culture_has_skirmisher_maa = {
 		# has_innovation = innovation_adaptive_militia #Goedendag - Later Era MAA
 
 		# Warcraft
-		# TODO
-		# Add mod MaAs here
+		has_cultural_parameter = unlock_maa_arachnathid
+		has_cultural_parameter = unlock_maa_bearpaws
+		has_cultural_parameter = unlock_maa_blademaster
+		has_cultural_parameter = unlock_maa_brute
+		has_cultural_parameter = unlock_maa_buccaneer
+		has_cultural_parameter = unlock_maa_candleholder
+		has_cultural_parameter = unlock_maa_dervishes
+		has_cultural_parameter = unlock_maa_ghoul
+		has_cultural_parameter = unlock_maa_gladiator
+		has_cultural_parameter = unlock_maa_savage
+		has_cultural_parameter = unlock_maa_snow_hunter
+		has_cultural_parameter = unlock_maa_swarm
+		has_cultural_parameter = unlock_maa_tidehunter
```

Comment:
These are not used because they are not set. So how it works in vanilla, there are tradition that have parameters.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/195980643-0c5f7119-c6f0-4dfa-9260-9e289d6c8003.png)

</details>
When you get tradition `has_cultural_parameter = unlock_maa_shomer` starts to return true.
In short, it returns true when some cultural MaA is unlocked.

Reply:
Is this not done 1.5 cultural adaptation? I was going of off the assumption that somewhere in wc_maa_traditions it was already done